app/controllers/user_posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UserPostsController < ApplicationController
```

Comment:
Please add `show` and `index` actions to the posts controller.

Reply:
resources :users, only: %i[index show]
  resources :users_posts, only: %i[index show]

Is already there ... 

Please kind change to approved